fusor-ember-cli/app/routes/subscriptions/management-application.js

```diff
@@ -89,10 +89,76 @@ export default Ember.Route.extend({
   },
 
   actions: {
+    willTransition(transition) {
+      if (this.modelFor('deployment').get('isNotStarted')) {
+        this.saveSma().catch(err => console.log(err));
+      }
+
+      return true;
+    },
+
     error(reason, transition) {
       // bubble up this error event:
       return true;
     }
-  }
+  },
+
+  saveSma() {
+    let controller = this.get('controller');
+    let deployment = this.modelFor('deployment');
+    let deploymentId = deployment.get('id');
+    let consumerUUID = deployment.get('upstream_consumer_uuid');
+    let isDisconnected = this.controllerFor('deployment').get('isDisconnected');
+
+    controller.set('isLoading', true);
+    controller.set('errorMsg', null);
+
+
+    return Ember.RSVP.hash({
```

Comment:
similar but not quite identical to [select-subscriptions route](https://github.com/fusor/fusor/blob/master/fusor-ember-cli/app/routes/subscriptions/select-subscriptions.js#L43)